.github/workflows/pr-check.yaml

```
@@ -150,14 +129,14 @@ jobs:
           echo "Version of @podman-desktop/tests-playwright to be used: $version"
           jq --arg version "$version" '.devDependencies."@podman-desktop/tests-playwright" = $version' package.json > package.json_tmp && mv package.json_tmp package.json
 
-      - name: Execute yarn in Bootc Extension
+      - name: Execute pnpm in Bootc Extension
         working-directory: ./podman-desktop-extension-bootc
-        run: yarn --frozen-lockfile
+        run: pnpm install
```

```suggestion
        run: pnpm install --no-frozen-lockfile
```
need to fetch a different version of test-playwright dependencies

https://github.com/redhat-developer/podman-desktop-redhat-account-ext/blob/main/.github/workflows/e2e-main.yaml#L117

---

Fixed.